setup.py

```diff
@@ -21,12 +21,13 @@
     long_description_content_type='text/x-rst',
     author="SeMI Technologies",
     author_email="hello@semi.technology",
-    packages=["semi", "semi.config", "semi.commands"],
+    packages=["semi", "semi.config", "semi.cloud", "semi.data", "semi.schema", "semi.classification"],
     py_modules=['cli'],
     python_requires='>=3.6',
     install_requires=[
         "weaviate-client>=2.1.0",
```

Comment:
Since we dropped the support for `weaviate-client` version 2.x.x. There were some bugs in the python-client so lets use the `3.7.0` version:
```python
        "weaviate-client>=3.7.0",
```